SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialStatics.h

```diff
@@ -108,6 +108,12 @@ class SPATIALGDK_API USpatialStatics : public UBlueprintFunctionLibrary
 	UFUNCTION(BlueprintCallable, Category = "SpatialOS")
 	static FColor GetInspectorColorForWorkerName(const FString& WorkerName);
 
+	/**
+	 * Returns the entity ID of a given actor, or 0 if we are not using spatial networking.
```

Comment:
> or 0

Should this be, or the empty string? And also maybe worth saying "if we are not using spatial networking or the Actor* argument is nullptr."?

Reply:
Arguably could return 0 since that's the invalid entity ID value used but I'm fine with either

Reply:
Good catch, this was written before it returned a string. Now fixed :)